python/tests/Examples.py

```diff
@@ -568,6 +570,33 @@ def test_simple_hypothesis_with_importance_with_cluster(self):
 
         self.dump_graph(g, "Simple hypothesis with importance with cluster")
 
+    def test_create_a_simple_cluster_with_handle(self):
```

Comment:
This test is throwing an Exception.`Exception: term 'handle' not in namespace 'https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/InterchangeOntology#'`
The look up table in aida_rdf_ontologies.py SEEDLING_TYPES_NIST needs to have the term "handle" added.